semver.md

```diff
@@ -319,6 +319,25 @@ Abbreviating "version" as "v" is often seen with version control. Example:
 `git tag v1.2.3 -m "Release version 1.2.3"`, in which case "v1.2.3" is a tag
 name and the semantic version is "1.2.3".
 
+### Is there a suggested egular expression (RegEx) to check a SemVer string?
```

Comment:
Should "egular" be "regular"?

Reply:
i wanted to comment `r?` but i thought it’d be too subtle

Reply:
@ljharb I get this joke.

Reply:
Pushed a commit to +r